C-git-commands.asc

```diff
@@ -577,7 +577,7 @@ Es gibt zudem eine ganze Reihe von Basisbefehlen, auf die wir in diesem Buch ges
 
 Zuerst begegnen wir `ls-remote` in <<ch06-github#_pr_refs>>, das wir zum Betrachten der Rohdaten auf dem Server verwenden.
 
-Wir verwenden `ls-files` in <<ch07-git-tools#_manual_remerge>>, <<ch07-git-tools#ref_rerere>> und <<ch07-git-tools#_the_index>>, um einen groben Einblick in Ihre Staging-Area zu erhalten.
+Wir verwenden `ls-files` in <<ch07-git-tools#_manual_remerge>>, <<ch07-git-tools#ref_rerere>> und <<ch07-git-tools#_the_index>>, um einen groben Einblick in deine Staging-Area zu erhalten.
```

Comment:
Wir verwenden `ls-files` in den Abschnitten <<ch07-git-tools#_manual_remerge>>, <<ch07-git-tools#ref_rerere>> und <<ch07-git-tools#_the_index>>, um einen direkteren bzw. roheren Blick darauf zu werfen, wie dein Staging-Bereich aussieht.